proxies/shared/resources/jsc/SetBackendCorrelationId.js

```diff
@@ -1,8 +1,12 @@
 const incomingCorrelationId = context.getVariable("request.header.x-correlation-id");
 const messageId = context.getVariable("messageid");
 
-if (incomingCorrelationId) {
+if (incomingCorrelationId && messageId) {
```

Comment:
Can `messageid` actually be null? I thought it was always generated by Apigee.

Reply:
Spoke with simon on a call to resolve this.